src/components/auth/EthWalletNewSelector.tsx

```diff
@@ -0,0 +1,133 @@
+    }, [onLogin, browserWallet, anchor])
+    const handleIncompatible = useCallback(() => {
+      track('click_incompatible_wallet')
+      setShowIncompatible(true)
+    }, [])
+
+    return (
+      <>
+        <LoginModal
+          open={open}
+          onClose={() => {
+            setShowIncompatible(false)
+            onClose && onClose()
+          }}
+          i18n={{
+            title: showIncompatible ? 'Create a Web3 wallet' : 'Create an account or sign in',
+            subtitle: showIncompatible ? (
+              <span>
+                Don’t have a compatible wallet? Don’t worry, creating one is <b>easy and free</b>. Choose one of the
+                providers below to start.
+              </span>
+            ) : (
+              <span>
+                Connect with one of our supported <b>Web3 wallet</b> providers to get started.
+              </span>
+            ),
+            error: ''
+          }}
+          className="login-new-modal"
+        >
+          <LoginModal.Option type={browserWallet || LoginModalOptionType.METAMASK} onClick={handleLoginInjected} />
+          <LoginModal.Option type={LoginModalOptionType.FORTMATIC} onClick={handleLoginFortmatic} />
+          {!showIncompatible && (
+            <LoginModal.Option type={LoginModalOptionType.WALLET_CONNECT} onClick={handleLoginWalletConnect} />
+          )}
+          {!showIncompatible && (
+            <LoginModal.Option type={LoginModalOptionType.WALLET_LINK} onClick={handleLoginWalletLink} />
+          )}
+          {!showIncompatible && (
+            <div className="dcl option incompatible" onClick={handleIncompatible}>
+              <div className="image"></div>
+              <div className="info">
+                <div className="title">I don't have a compatible wallet</div>
+              </div>
+            </div>
+          )}
+          {loading && (
+            <div className="loader-background">
+              <Loader active={loading} provider={provider} size="massive" />
+              <EthConnectAdvice provider={provider} style={{ marginTop: '27px' }} />
+              <div style={{ marginTop: '22px' }}>- or -</div>
+              <Button onClick={onCancelLogin} loading={canceling} style={{ marginTop: '28px' }}>
+                cancel login
+              </Button>
+            </div>
+          )}
+
+          {showIncompatible && (
+            <div className="login-new-modal__incompatible-description">
+              <h2>FAQ</h2>
+              <h3>What is a Web3 wallet?</h3>
+              <p>A Web3 wallet is a type of crypto wallet. It stores cryptocurrency and NFTs.</p>
+              <h3>Why do I need a Web3 wallet?</h3>
+              <p>
+                Your wallet is your Decentraland account. It stores your digital assets and progress and lets you
+                connect from more than one device.
+              </p>
+              <h3>Do I need to pay?</h3>
+              <p>No, Decentraland is free to use.</p>
```

Comment:
Isn't this question referring to the FAQ of the Web3 wallets? If so, this should respond the question "Do I need to pay to use a web3 wallet?"
```suggestion
              <h3>Do I need to pay?</h3>
              <p>No, there are plenty of Web3 free wallets and Decentraland is free to use.</p>
```